apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -1,18 +1,60 @@
 {% extends "base.html" %}
 {% block content %}
 <style>
+  .container {
+    margin-left: 44.5px;
+    margin-right: 44.5px;
+    padding-left: 15px;
+    padding-right: 15px;
+  }
```

Comment:
**Unexpected.** Please help me understand its goal.
<sub>The `.container` already has complex spacing ([mostly from Bootstrap](https://getbootstrap.com/docs/4.0/layout/overview/#containers)) that changes per screen width.</sub>